### PR TITLE
fix(indexer): Add name to toml config

### PR DESCRIPTION
**Pull request by roninjin10**
- forgot to include name in the toml config

### CODE REVIEW

**Comment by roninjin10** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #5943** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5943" target="_blank">Graphite</a> 
    * **PR #5944** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5944" target="_blank">Graphite</a> 
      * **PR #5945** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5945" target="_blank">Graphite</a> 
        * **PR #5946** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5946" target="_blank">Graphite</a> 
          * **PR #5947** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5947" target="_blank">Graphite</a> 
            * **PR #5969** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5969" target="_blank">Graphite</a>  👈
            * **PR #5948** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5948" target="_blank">Graphite</a> 
              * **PR #5950** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5950" target="_blank">Graphite</a> 
                * **PR #5951** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5951" target="_blank">Graphite</a> 
                  * **PR #5952** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5952" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/5969?utm_source=stack-comment).

File: indexer/config/config_test.go [hunk 1]
```diff
@@ -26,6 +26,7 @@ func TestLoadConfig(t *testing.T) {
 		port = 5432
 		user = "postgres"
 		password = "postgres"
+	  name = "indexer"
```
Comment (tynes) [inline indexer/config/config_test.go:29]: spacing looks off

Comment (roninjin10) [inline indexer/config/config_test.go:29]:
```suggestion
	    name = "indexer"
```

File: indexer/cli/cli.go [hunk 2]
```diff
@@ -32,6 +35,11 @@ func runIndexer(ctx *cli.Context) error {
 	return nil
 }
 
+// Maybe make NewDB take a config.DBConfig instead of a string in future cleanup
+func getDsn(dbConf config.DBConfig) string {
+	return fmt.Sprintf("postgresql://%s:%s@%s:%s/%s", dbConf.User, dbConf.Password, dbConf.Host, dbConf.Port, dbConf.Name)
```
Comment (roninjin10) [inline indexer/cli/cli.go:40]:
```suggestion
	return fmt.Sprintf("postgresql://%s:%s@%s:%s/%s", dbConf.User, dbConf.Password, dbConf.Host, strconv.Itoa(dbConf.Port), dbConf.Name)
```
Noticed the linter thorwing a warning for this

File: indexer/cli/cli.go [hunk 3]
```diff
@@ -3,8 +3,11 @@ package cli
 import (
```
Comment (roninjin10) [inline indexer/cli/cli.go:3]: Leaving this comment open reminding myself to squash the 4 commits into 1

**Review by tynes — APPROVED** [review]

**Review by hamdiallam — APPROVED** [review]

**Pull request merged**